release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Bug fixes to logout of RSA Archer API at end of actions and return success for 0 records of 'list tickets' action
```

Comment:
As multiple bugs are fixed, can we mention each fix separately?
* Bug fix to logout of RSA Archer API at the end of actions
* Bug fix to return success for 0 records of 'list tickets' action

Reply:
Done